src/domains/citizen-auth-app/08_session_manager.tf

```diff
@@ -36,6 +36,11 @@ data "azurerm_key_vault_secret" "app_backend_ALLOWED_CIE_TEST_FISCAL_CODES" {
   key_vault_id = data.azurerm_key_vault.kv_common.id
 }
 
+data "azurerm_key_vault_secret" "app_backend_TEST_LOGIN_PASSWORD" {
+  name         = "appbackend-TEST-LOGIN-PASSWORD"
+  key_vault_id = data.azurerm_key_vault.kv_common.id
+}
+
```

Comment:
should we add it to our custom kv? I would also rename it to something like `session-manager-TEST-LOGIN-PASSWORD`

Reply:
Fixed in 13f25fc